apm-sniffer/config/agent.config

```diff
@@ -38,6 +38,15 @@ agent.service_name=${SW_AGENT_NAME:Your_ApplicationName}
 # SkyWalking team may ask for these files in order to resolve compatible problem.
 # agent.is_open_debugging_class = ${SW_AGENT_OPEN_DEBUG:true}
 
+# If true, SkyWalking agent will cache all instrumented classes files to memory or disk files (decided by class cache mode),
```

Comment:
Both these are very advanced config, please keep these two in the documents only. Most users even can't understand what this is about, causing unnecessary confusion. I would like to recommend you adding a FAQ doc, including the error you were facing, and link to the agent set up the document and the config.

Reply:
Add class cache faq doc: `Compatible with other javaagent bytecode processing`